docs/DeveloperGuide.md

```diff
@@ -1060,6 +1058,31 @@ testers are expected to do more *exploratory* testing.
     (where incomplete details are given for the applicant being added).
     Expected: Error messages displaying the cause of error is shown in the result display.  
 
+### Editing an applicant
+
+1. Editing an applicant while all applicants are being shown
+
+  1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
```

Comment:
I think `list` is not a necessary prerequisite because users should be able to `edit` after any command?